roles/contiv_network/tasks/main.yml

```diff
@@ -41,13 +29,20 @@
 - name: install contivctl
   shell: source /etc/profile.d/00golang.sh && go get github.com/contiv/contivctl
 
-- name: install docker-compose
-  shell: pip install docker-compose
+- yum: name=python-pip
 
-- name: install contiv-compose
-  shell: >
-     source /etc/profile.d/00golang.sh && \
-     go get github.com/contiv/libcompose && \
-     cd $GOPATH/src/github.com/contiv/libcompose && \
-     make binary && \
-     ln -s `pwd`/bundles/libcompose-cli_linux-amd64 $GOPATH/bin/contiv-compose
+- pip: name=docker-compose state=latest extra_args=--upgrade
+
+- name: copy environment file for netmaster
```

Comment:
it can't be started before netplugin, which was the reasoning for moving it. I will move the comment(s).